opa/stackable/bin/process-logs

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+#
+# This file was created by the Stackable developers.
+#
+# Usage: process-logs <options>
+# Options:
+#   --console-log-level <log-level>
+#   --file-log-level <log-level>
+#   --decision-log-level <log-level>
+#   --server-log-level <log-level>
+#   --opa-rolling-log-file-size-bytes <maximum-file-size>
+#   --opa-rolling-log-files <number-of-log-files>
+#   --stackable-log-dir <path-to-log-dir>
+#   --container-name <container-name>
+#
+# Processes incoming log messages. Log messages are filtered by the set log levels
+# and forwarded to the output destinations console and/or file.
+#
+
+parse_args() {
+  while true; do
+    case $1 in
+        --console-log-level)
+            shift
+            CONSOLE_LEVEL=$1
+            ;;
+        --file-log-level)
+            shift
+            FILE_LEVEL=$1
+            ;;
+        --decision-log-level)
+            shift
+            DECISION_LEVEL=$1
+            ;;
+        --server-log-level)
+            shift
+            SERVER_LEVEL=$1
+            ;;
+        --opa-rolling-log-file-size-bytes)
+            shift
+            OPA_ROLLING_LOG_FILE_SIZE_BYTES=$1
+            ;;
+        --opa-rolling-log-files)
+            shift
+            OPA_ROLLING_LOG_FILES=$1
+            ;;
+        --stackable-log-dir)
+            shift
+            STACKABLE_LOG_DIR=$1
+            ;;
+        --container-name)
+            shift
+            CONTAINER_NAME=$1
+            ;;
+        *)
+            break
+            ;;
+    esac
+    shift
+  done
```

Comment:
Wouldn't it be simpler to just pass these as envvars directly?

Reply:
True, it would be simpler. Given that it is done like this already, is it a show-stopper?